tests/user_test.go

```diff
@@ -1,76 +1,125 @@
+			Name: &userName,
+		},
+	)
+
+	retrievedUser, _ := client.GetUser(user.ID)
+
+	assert.Equal(reflect.TypeOf(&easypost.User{}), reflect.TypeOf(retrievedUser))
+	assert.True(strings.HasPrefix(retrievedUser.ID, "user_"))
+}
+
+func (c *ClientTests) TestUserRetrieveMe() {
 	client := c.ProdClient()
 	assert := c.Assert()
-	opts := &easypost.UserOptions{Name: easypost.StringPtr("Go Test User")}
-	user, err := client.CreateUser(opts)
-	if assert.NoError(err) {
-		assert.NotEmpty(user.ID)
-		if user, err = client.GetUser(user.ID); assert.NoError(err) {
-			assert.Equal("Go Test User", user.Name)
-		}
-		*opts.Name = "Go Testing User"
-		opts.ID = user.ID
-		if user, err = client.UpdateUser(opts); assert.NoError(err) {
-			if user, err = client.GetUser(user.ID); assert.NoError(err) {
-				assert.Equal("Go Testing User", user.Name)
-			}
-		}
-		assert.NoError(client.DeleteUser(user.ID))
-	}
-
-	opts.Password = easypost.StringPtr("super-secret-password")
-	opts.PasswordConfirmation = opts.Password
-	_, err = client.CreateUser(opts)
-	if assert.IsType((*easypost.APIError)(nil), err) {
-		err := err.(*easypost.APIError)
-		assert.Equal(
-			"Child users must be created via the API and must not contain an email or password.",
-			err.Message,
-		)
-		assert.Equal("USER.INVALID", err.Code)
-	}
+
+	user, _ := client.RetrieveMe()
+
+	assert.Equal(reflect.TypeOf(&easypost.User{}), reflect.TypeOf(user))
+	assert.True(strings.HasPrefix(user.ID, "user_"))
+}
+
+func (c *ClientTests) TestUserUpdate() {
+	client := c.ProdClient()
+	assert := c.Assert()
+
+	user, _ := client.RetrieveMe()
+
+	test_phone := "5555555555"
+
+	updatedUser, _ := client.UpdateUser(
+		&easypost.UserOptions{
+			ID:          user.ID,
+			PhoneNumber: &test_phone,
+		},
+	)
+
+	assert.Equal(reflect.TypeOf(&easypost.User{}), reflect.TypeOf(updatedUser))
+	assert.True(strings.HasPrefix(updatedUser.ID, "user_"))
+	assert.Equal(test_phone, updatedUser.PhoneNumber)
+}
+
+func (c *ClientTests) TestUserUpdateBrand() {
+	client := c.ProdClient()
+	assert := c.Assert()
+
+	color := "#123456"
+
+	user, _ := client.RetrieveMe()
+
+	brand, err := client.UpdateBrand(
+		map[string]interface{}{
+			"color": color,
+		},
+		user.ID,
+	)
+
+	fmt.Println(err)
+
+	assert.Equal(reflect.TypeOf(&easypost.Brand{}), reflect.TypeOf(brand))
+	// assert.True(strings.HasPrefix(brand.ID, "brd_")) Brand does not have ID, this will be fixed when we patch it
+	assert.Equal(color, brand.Color)
+}
+
+/* Due to our inability to create child users securely, we must also skip deleting them as we cannot replace the deleted ones easily.
+func (c *ClientTests) TestUserDelete() {
+	client := c.ProdClient()
+	require := c.Require()
+
+	userName := "Test User"
+
+	user, _ := client.CreateUser(
+		&easypost.UserOptions{
+			Name: &userName,
+		},
+	)
+
+	err := client.DeleteUser(user.ID)
+
+	require.NoError(err)
+}
+*/
+
+/* API keys are returned as plaintext, do not run this test.
+func (c *ClientTests) TestUserApiKeysAll() {
```

Comment:
Do not use large block comments to skip tests, use the `t.Skip()` function as mentioned previously